src/dbnode/persist/fs/seek_manager.go

```diff
@@ -496,6 +500,51 @@ func (m *seekerManager) updateOpenLeaseHotSwapSeekers(
 	return wg, updateOpenLeaseResult, nil
 }
 
+func (m *seekerManager) RelinquishShard(
+	namespace ident.ID,
```

Comment:
I'm actually a little torn about whether `namespace` should be part of this interface. I guess technically it gives us more control but for all use-cases I can think of we'll want to do it for all namespaces. @robskillington thoughts?

Reply:
I removed `namespace` for now